docs/introduction.rst

```diff
@@ -21,14 +21,17 @@ user to get most large datasets up and running by hours, and in some cases days.
 What data tasks does the Retriever handle
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Data Retriever handles a number of common tasks including: 1) creating
-the underlying database structures, including automatically determining the data
-types; 2) downloading the data; 3) transforming data into appropriately
-normalized forms for database management systems (e.g., "wide" data into "long"
-data and splitting tables into proper sub-tables to reduce duplication); 4)
-converting heterogeneous null values (e.g., 999.0, -999, NaN) into standard null
-values; 5) combining multiple data files into single tables; and 6) placing all
-related tables in a single database or schema.
+The Data Retriever handles a number of common tasks including:
+1. Creating the underlying database structures, including automatically
+ determining the data types
```

Comment:
I think there's a new line added here which should not be there, causing the formatting to be a little off. Same thing for 3, 4, and 5.